glass/observations.py

```diff
@@ -78,16 +77,16 @@ def vmap_galactic_ecliptic(
     m[hp.query_strip(nside, *galactic)] = 0
     m = hp.Rotator(coord="GC").rotate_map_pixel(m)
     m[hp.query_strip(nside, *ecliptic)] = 0
-    return hp.Rotator(coord="CE").rotate_map_pixel(m)  # type: ignore[no-any-return]
+    return hp.Rotator(coord="CE").rotate_map_pixel(m)
 
 
 def gaussian_nz(
-    z: npt.NDArray[typing.Any],
-    mean: npt.ArrayLike,
-    sigma: npt.ArrayLike,
+    z: npt.NDArray[np.float64],
+    mean: npt.NDArray[np.float64],
+    sigma: npt.NDArray[np.float64],
```

Comment:
these should accept `float` since they are in practice often called with single float arguments